source/type/type.h

```diff
@@ -1129,6 +1136,27 @@ std::string get_admin_name(const T* v) {
     }
     return admin_name;
 }
+
+struct JourneyPatternKey {
+    bool is_frequence = false;
+    Route* route = nullptr;
+    PhysicalMode* physical_mode = nullptr;
+    CommercialMode* commercial_mode = nullptr;
+    hasOdtProperties odt_properties{};
+    std::string name{};
+    std::string uri{};
```

Comment:
well we need all the JP data but the journey pattern points (and we want stop points instead) no ?